### ISSUE TITLE
Wrap CRIs in CBOR byte strings

### ISSUE THREAD

**Issue by ektrah** [core-wg/coral#63]
As discussed in a couple of places (like [here](https://github.com/core-wg/coral/issues/43#issuecomment-568245426) and [here](https://github.com/core-wg/coral/issues/2#issuecomment-573118030)), we should consider wrapping CRIs in CBOR byte strings.

**Comment by chrysn** [comment]
> wrapping CRIs in CBOR byte strings

yes. it's a byte more for references >23 long (as they'd rarely have 23 elements), but well spent to save processing.
